frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/entity/Features.java

```diff
@@ -1,28 +1,9 @@
 package uk.gov.di.authentication.frontendapi.entity;
 
-import com.google.gson.annotations.Expose;
-import com.google.gson.annotations.SerializedName;
-
 public class Features {
 
-    @SerializedName("updatePasswordHintTextVersion")
```

Comment:
Could we remove this entire class now?

Reply:
Thanks @BeckaL - I thought the same and asked about this before raising the PR. The view was that it would be beneficial to retain the facility. What I should have done was update the PR to reflect this (I've done that now) I'll send you the Slack thread 👍🏽

Reply:
Ah cool, makes sense